packages/webapp/pages/account/notifications.tsx

```diff
@@ -378,6 +399,14 @@ const AccountNotificationsPage = (): ReactElement => {
         >
           Community updates
         </Checkbox>
+        <Checkbox
+          name="following"
+          data-testid="following-switch"
+          checked={followingEmail}
+          onToggleCallback={onToggleFollowingEmail}
+        >
+          Updates from followed sources, squads and users
```

Comment:
We might have to check with @capJavert if we want to change the copy (for the time being to:)
`Updates from followed users`

Reply:
Updated